src/index.js

```diff
@@ -21,6 +24,17 @@ const run = async () => {
   console.log("->> Generating markdown…");
   const markdown = await generateMarkdownReport(results);
 
+  // Expose the markdown to an Action output
+  // https://github.community/t/set-output-truncates-multiline-strings/16852
+  const escaped_markdown = markdown.replace(/\%/g,'%25').replace(/\n/g,'%0A').replace(/\r/g,'%0D')
```

Comment:
Could you update `escaped_markdown` to be camelCase?

Reply:
Done!